.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,75 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", release/0.26.2, release/0.32.2 ]
```

Comment:
I don't think we need to also point release branches here. After all, release branches will be expected to contain only cherry-picked commits that have been merged in mailine first. What do you think?

Alternatively, is something like `release/*` a viable option in GHA?

Reply:
Agree I left them from the previous PR but I think we don't need the release branches since they are just cherry-picked commits, I will remove them, thank you :rocket: